Formula/open-zwave.rb

```diff
@@ -1,132 +1,32 @@
 class OpenZwave < Formula
   desc "Library for selected Z-Wave PC controllers"
-  homepage "http://www.openzwave.com"
-  url "http://old.openzwave.com/downloads/openzwave-1.2.919.tar.gz"
-  sha256 "473229f3dd3d6b260e6584b17e5c5f2e09e61805f89763f486a9f7aa2b4181ba"
+  homepage "http://www.openzwave.net/"
+  url "https://github.com/OpenZWave/open-zwave/archive/v1.4.tar.gz"
+  sha256 "efb54adf0cda12f76fcb478f7cf3b7d108b5daf38d60b4410cca7600aa072e83"
+  head "https://github.com/OpenZWave/open-zwave.git"
```

Comment:
are you planning to use `--HEAD` personally?

Reply:
Yep. 
openzwave includes xml definitions for devices and those are updated quite often.
https://github.com/OpenZWave/open-zwave/tree/master/config

Reply:
OK.